actions/parse-push/src/main.ts

```diff
@@ -145,36 +163,58 @@ async function parseCommit(commit: any): Promise<Commit> {
     }
 }
 
+type WebhookPayloadPush = EventPayloads.WebhookPayloadPush
+
+function isPushContext(
+    _payload: WebhookPayload
+): _payload is Omit<WebhookPayloadPush, 'commits'> & {
+    commits: readonly RemoteCommit[]
+} {
+    return context.eventName === 'push'
+}
```

Comment:
This is really hard to read and understand with all these line breaks.

Reply:
It's definitely not ideal. The only way around it is to _extract_ the type into a `type Foo`

Reply:
or fix the printWidth :P

Reply:
Changing `printWidth` will only _maybe_ fix "this" instance, but not any that's longer, and will weirdly un-linebreak other lines for no good reason.